### PR TITLE
updated README.md and added CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by chrislomonico**
Per [161](https://github.com/agera-edc/MinimumViableDataspace/issues/161)
- Added `CONTRIBITUTING.md`
- Updated `README.md`

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,237 @@
...
+before raising an issue.
+
+Please note that if an issue covers a topic or the response to a question that may be interesting 
+for other developers or contributors, or for further discussions, it should be converted to a 
+discussion and not be closed.
+
+### Adhere to Coding Style Guide
+
+We aim for a coherent and consistent code base, thus the coding style detailed in the 
+[styleguide](styleguide.md) should be followed.
```
Comment (cpeeyush) [inline CONTRIBUTING.md:84]: styleguide.md - broken link.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,237 @@
...
+
+### Submit a Pull Request
+
+In addition to the contribution guideline made available in the 
+[Eclipse project handbook](https://www.eclipse.org/projects/handbook/#contributing),
+we would appreciate if your pull request applies to the following points:
+
+* Conform to [Pull-Request Etiquette](pr_etiquette.md)
```
Comment (cpeeyush) [inline CONTRIBUTING.md:92]: broken link.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,237 @@
...
+  include a ["Signed-off-by" entry](https://wiki.eclipse.org/Development_Resources/Contributing_via_Git).
+  
+* Add meaningful tests to verify your submission acts as expected.
+
+* Where code is not self-explanatory, add documentation providing extra clarification.
+
+* Add documentation files to new modules. See [here](#add-documentation) for more details.
+
+* Add relevant changes (e.g., no typo fixes, updated readme files, fixes of stuck test) to the 
+  [changelog](CHANGELOG.md). If these refer to a new feature, add this to the `Overview` section
```
Comment (cpeeyush) [inline CONTRIBUTING.md:133]: broken link

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,237 @@
...
+
+* Add relevant changes (e.g., no typo fixes, updated readme files, fixes of stuck test) to the 
+  [changelog](CHANGELOG.md). If these refer to a new feature, add this to the `Overview` section 
+  and add your changes to the `Detailed Changes` section according to the rules documented on 
+  <http://keepachangelog.com>. Include more information via linking to existing pull requests,
+  issues, or discussions.
+
+* If a new module has been added or a significant part of the code has been changed and you should 
+  or want to be seen as the contact person for any further changes, please add appropriate 
+  information to the [CODEOWNERS](https://github.com/eclipse-dataspaceconnector/DataSpaceConnector/blob/main/CODEOWNERS)
```
Comment (cpeeyush) [inline CONTRIBUTING.md:140]: link needs to be updated - currently it is pointing to EDC repo.

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,237 @@
...
+  issues, or discussions.
+
+* If a new module has been added or a significant part of the code has been changed and you should 
+  or want to be seen as the contact person for any further changes, please add appropriate 
+  information to the [CODEOWNERS](https://github.com/eclipse-dataspaceconnector/DataSpaceConnector/blob/main/CODEOWNERS) 
+  file. You can find instructions on how to do this at <https://help.github.com/articles/about-codeowners/>.
+  Please note that this file does not represent all contributions to the code. What persons and organizations
+  actually contributed to each file can be seen on GitHub and is documented in the license headers.
+
+* PR descriptions should use the current [PR template](.github/PULL_REQUEST_TEMPLATE.md)
```
Comment (cpeeyush) [inline CONTRIBUTING.md:145]: PR template - broken link

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -0,0 +1,237 @@
...
+Be sure to use the same email address in your Eclipse Account that you intend to use when you commit 
+to GitHub.
+
+## How to Contribute
+
+### Discuss
+
+If you want to share an idea to further enhance
+the project or discuss potential use cases, please feel free to create a discussion at the
+[GitHub Discussions page](https://github.com/eclipse-dataspaceconnector/DataSpaceConnector/discussions).
```
Comment (cpeeyush) [inline CONTRIBUTING.md:55]: This link points to EDC repo discussion page. But I think it should point to this repo discussion page. Right ?

File: CONTRIBUTING.md [hunk 7]
```diff
@@ -0,0 +1,237 @@
...
+otherwise [create a new issue](#create-an-issue). 
+
+### Create an Issue
+
+If you have identified a bug or want to formulate a working item that you want to concentrate on, 
+feel free to create a new issue at our project's corresponding
+[GitHub Issues page](https://github.com/eclipse-dataspaceconnector/DataSpaceConnector/issues/new).
+
+Before doing so, please consider searching for potentially suitable
+[existing issues](https://github.com/eclipse-dataspaceconnector/DataSpaceConnector/issues?q=is%3Aissue+is%3Aopen).
```
Comment (cpeeyush) [inline CONTRIBUTING.md:67]: This link should point to DataDashBoard repo issues. Currently it is pointing to EDC repo issues page.

File: CONTRIBUTING.md [hunk 8]
```diff
@@ -0,0 +1,237 @@
...
+
+We also use [GitHub's default label set](https://docs.github.com/en/issues/using-labels-and-milestones-to-track-work/managing-labels)
+extended by custom ones to classify issues and improve findability.
+
+If an issue appears to cover changes that will have a (huge) impact on the code base and needs to 
+first be discussed, or if you just have a question regarding the usage of the software, please 
+create a [discussion](https://github.com/eclipse-dataspaceconnector/DataSpaceConnector/discussions)
```
Comment (cpeeyush) [inline CONTRIBUTING.md:74]: This link points to EDC repo discussion page. But I think it should point to this repo discussion page. Right ?

File: CONTRIBUTING.md (hunk 2 above)
Comment (chrislomonico) [reply CONTRIBUTING.md:89]: added pr_etiquette

File: CONTRIBUTING.md (hunk 3 above)
Comment (chrislomonico) [reply CONTRIBUTING.md:130]: added CHANGELOG.md

File: CONTRIBUTING.md (hunk 4 above)
Comment (chrislomonico) [reply CONTRIBUTING.md]: updated link to point to current repo

File: CONTRIBUTING.md (hunk 1 above)
Comment (chrislomonico) [reply CONTRIBUTING.md:81]: added styleguid

File: CONTRIBUTING.md (hunk 5 above)
Comment (chrislomonico) [reply CONTRIBUTING.md:142]: added template with updated content

File: CONTRIBUTING.md (hunk 6 above)
Comment (chrislomonico) [reply CONTRIBUTING.md]: updated link

File: docs/templates/README.md [hunk 9]
```diff
@@ -0,0 +1,34 @@
...
+Find all provided documentation templates in this folder. Please note that the _italic text 
+and sentences_ should be removed. Feel free to add additional sections and subsections, however, make sure
+that at least the sections of the templates marked as "mandatory" are filled.
+
+## Decision Records
+
+[Link](decision-record.md) to template.
+
+Each decision record should be put in an appropriate folder that is following a naming pattern: 
+`YYYY-MM-DD-title-of-decision-record`. This should be located at the [decision record folder](../developer/decision-records)
```
Comment (cpeeyush) [inline docs/templates/README.md:12]: nit: broken link. Pls ignore if we do not need this one here or we have some other ways to cover it from upstream.

Just adding the comment to highlight what is missing.

File: docs/templates/README.md [hunk 10]
```diff
@@ -0,0 +1,34 @@
...
+and contain all relevant files, at least a filled-out template named `README.md` and any additional images.
+
+As of now, every merged decision record is in state `accepted`. Please make sure to add a comment to
+a decision record that replaces a previous one with adding a hint: `superseded by [...]`.
+
+## Extensions
+
+[Link](extension.md) to template.
+
+Every module located [in the extensions folder](../../extensions) has to provide documentation regarding its
```
Comment (cpeeyush) [inline docs/templates/README.md:22]: nit: broken link. Pls ignore if we do not need this one here or we have some other ways to cover it from upstream.

Just adding the comment to highlight what is missing.

**Review by cpeeyush — APPROVED** [review]
Approved with suggestions.

File: docs/templates/README.md (hunk 9 above)
Comment (chrislomonico) [reply docs/templates/README.md:12]: removed sections specific to EDC, included templates

File: docs/templates/README.md (hunk 10 above)
Comment (chrislomonico) [reply docs/templates/README.md]: removed edc specific section from file

**Review by algattik — APPROVED** [review]

**Pull request merged**